ob1k-cache/pom.xml

```diff
@@ -78,5 +77,16 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>net.jpountz.lz4</groupId>
+      <artifactId>lz4</artifactId>
+      <version>1.3.0</version>
```

Comment:
please move both versions to main pom dependency management
also, I'm thinking if we're doing here right that we enforce that dependency, but not a blocker for now..